x/incentives/keeper/pot_test.go

```diff
@@ -331,3 +331,48 @@ func (suite *KeeperTestSuite) TestPerpetualPotOperations() {
 	rewardsEst = suite.app.IncentivesKeeper.GetRewardsEst(suite.ctx, lockOwner, []lockuptypes.PeriodLock{}, 100)
 	suite.Require().Equal(sdk.Coins(nil), rewardsEst)
 }
+
+func (suite *KeeperTestSuite) TestNoLockPotDistribution() {
```

Comment:
Should we have a test for the non-perpetual variant as well?

The test for the perpetual variant LGTM

Reply:
Added test for non perpetual pot distribution for no locks